### DISCUSSION TITLE
About the parameter settings and principles of the Spurt phase unwrapping method.

### DISCUSSION THREAD

**Discussion by ichaochao9** [isce-framework/dolphin#388] [Q&A]
Hello, developers,

I created the dolphin_config.yaml file to run dolphin. I don't understand the meanings of some parameters in the spurt unwrapping method settings. In the following figure, how should I understand the meanings corresponding to the default values of the parameters ‘t_cost_type’ and ‘t_cost_scale’？
![WechatIMG8](https://github.com/user-attachments/assets/d1c5c5c5-196b-4e03-935c-0abe2af3e8c5)

Could you please briefly explain or provide some references to help me understand the principle of the spurt unwrapping method?

**Comment by scottstanie** [comment]
Hi, the `spurt` parameters were copied from its own workflow parameters and defaults.
Since `spurt` is quite new/under active development, it could be more appropriate to ask in the [spurt discussions](https://github.com/isce-framework/spurt/discussions). I know the documentation is [currently being built out](https://github.com/isce-framework/spurt/pull/36)

**Reply by ichaochao9** [reply]
Thank you for your response, I will resume my questions in the [spurt discussions](https://github.com/isce-framework/spurt/discussions).
